packages/client/src/exampleTypes.js

```diff
@@ -194,7 +194,8 @@ export default F.stampKey('type', {
     },
     defaults: {
       field: null,
-      range: [0, 1000],
+      min: 0,
+      max: 1000,
       steps: [0, 1000, 5000, 10000, 25000, 50000, 1000000],
```

Comment:
**blocker:** Remove default steps

I know this was here before but there's no reasonable default for `steps`. It's totally dependent on the specific data you're querying and it may even be dynamically calculated in the future.